modules/simple_forms_api/app/controllers/simple_forms_api/v1/scanned_form_uploads_controller.rb

```diff
@@ -98,6 +98,16 @@ def perform_pdf_upload(location, file_path, metadata)
           upload_url: location
         )
       end
+
+      def check_for_changes
+        in_progress_form = InProgressForm.form_for_user('FORM-UPLOAD-FLOW', @current_user)
+        if in_progress_form
+          prefill_data_service = SimpleFormsApi::PrefillDataService.new(prefill_data: in_progress_form.form_data,
+                                                                        form_data: params[:form_data],
+                                                                        form_id: params[:form_number])
+          prefill_data_service.check_for_changes
+        end
+      end
```

Comment:
Since `in_progress_form` is only used in the if statement condition, you could not worry about the assignment and check like so:

```suggestion
      def check_for_changes
        return unless InProgressForm.form_for_user('FORM-UPLOAD-FLOW', @current_user)

        prefill_data_service = SimpleFormsApi::PrefillDataService.new(prefill_data: in_progress_form.form_data,
                                                                        form_data: params[:form_data],
                                                                        form_id: params[:form_number])
        prefill_data_service.check_for_changes
      end
```